tests/common/config/common/test_template_handler.py

```diff
@@ -169,31 +167,3 @@ def test_to_float():
     assert 0.0 == _TemplateHandler._to_float("0")
     assert -2.1 == _TemplateHandler._to_float("-2.1")
     assert 156165.3 == _TemplateHandler._to_float("156165.3")
-
-
-def test_to_scope():
-    with pytest.raises(InconsistentEnvVariableError):
-        _TemplateHandler._to_scope("okhds")
-    with pytest.raises(InconsistentEnvVariableError):
-        _TemplateHandler._to_scope("plop")
-
-    assert Scope.GLOBAL == _TemplateHandler._to_scope("global")
-    assert Scope.GLOBAL == _TemplateHandler._to_scope("GLOBAL")
-    assert Scope.SCENARIO == _TemplateHandler._to_scope("SCENARIO")
-    assert Scope.CYCLE == _TemplateHandler._to_scope("cycle")
-
-
-def test_to_frequency():
-    with pytest.raises(InconsistentEnvVariableError):
-        _TemplateHandler._to_frequency("okhds")
-    with pytest.raises(InconsistentEnvVariableError):
-        _TemplateHandler._to_frequency("plop")
-
-    assert Frequency.DAILY == _TemplateHandler._to_frequency("DAILY")
-    assert Frequency.DAILY == _TemplateHandler._to_frequency("Daily")
-    assert Frequency.WEEKLY == _TemplateHandler._to_frequency("weekly")
-    assert Frequency.WEEKLY == _TemplateHandler._to_frequency("WEEKLY")
-    assert Frequency.MONTHLY == _TemplateHandler._to_frequency("Monthly")
-    assert Frequency.MONTHLY == _TemplateHandler._to_frequency("MONThLY")
-    assert Frequency.QUARTERLY == _TemplateHandler._to_frequency("QuaRtERlY")
-    assert Frequency.YEARLY == _TemplateHandler._to_frequency("Yearly")
```

Comment:
Should we keep this test somewhere else?
With the `_replace_template()` method like the tests above

Reply:
Hum!

I removed the tests because I removed the methods `to_scope` and `to_frequency` that were never called. The replace_template method never calls the to_scope or to_frequency methods.

Today, we cannot set scope and frequency as an env variable. The following does not work.
```
Config.configure_datanode("my_id", scope="ENV[MY_ENV_VAR")
``` 

If we want to change that, we can create a dedicated issue. In any case, the `to_scope` and `to_frequency` methods should be replaced by a generic code that handles all generic cases.

Reply:
Agree.

In any case, the tests in `tests/core/config/test_config_serialization.py` already test the serialization of Scope and Frequency.

We can resolve this comment.